extension/WebJobs.Extensions.RabbitMQ.Tests/RabbitMQTriggerBindingTests.cs

```diff
@@ -54,13 +63,109 @@ public void Verify_BindingDataContract_Values()
             ["Body"] = body,
             ["Exchange"] = eventArgs.Exchange,
             ["BasicProperties"] = eventArgs.BasicProperties,
+            ["MessageActions"] = messageActions,
         };
 
-        IReadOnlyDictionary<string, object> actualContract = RabbitMQTriggerBinding.CreateBindingData(eventArgs);
+        IReadOnlyDictionary<string, object> actualContract = RabbitMQTriggerBinding.CreateBindingData(eventArgs, messageActions);
 
         foreach (KeyValuePair<string, object> item in actualContract)
         {
             Assert.Equal(data[item.Key], item.Value);
         }
     }
+
+    [Fact]
+    public void RabbitMQTriggerAttribute_ManualAck_DefaultsToFalse()
+    {
+        var attribute = new RabbitMQTriggerAttribute("test-queue");
+        bool manualAck = attribute.ManualAck;
+        Assert.False(manualAck, "ManualAck should default to false.");
+    }
+
+    [Fact]
+    public void RabbitMQTriggerBinding_CreateBindingData_IncludesRabbitMQMessageActions()
+    {
+        ulong deliveryTag = 1;
```

Comment:
Isn't `Verify_BindingDataContract_Values` already covering this case after your modifications?